src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1275,9 +1276,35 @@ def add_waveforms(
             if prop not in sorting_copy.get_property_keys():
                 sorting_copy.set_property(prop, tm[prop])
 
+    # find electrode table region corresponding to the recording
```

Comment:
I just realized that recording above is optional but this call will fail if recording is None.